components/dashboard/src/components/BillingAccountSelector.tsx

```diff
@@ -62,27 +82,43 @@ export function BillingAccountSelector(props: { onSelected?: () => void }) {
                 <div>
                     <p>Bill all my usage to:</p>
                     <div className="mt-4 flex space-x-3">
```

Comment:
**issue(non-blocking)**: Could we wrap the cards and limit the number of entries per row as seen in https://github.com/gitpod-io/gitpod/issues/11498#issuecomment-1211851137 to avoid breaking the layout or overflowing the page container?

| Multiple billing accounts |
|-|
| <img width="1440" alt="Screenshot 2022-08-18 at 2 33 12 PM" src="https://user-images.githubusercontent.com/120486/185385284-e6de0f06-aaf3-4778-8f05-ba23eb044137.png"> |

Reply:
Good idea! I've changed the layout from a `flex` to a `grid`, because that's more appropriate:

| No team | One team | Two teams | Three teams | Four teams |
| --- | --- | --- | --- |
| <img width="1436" alt="Screenshot 2022-08-18 at 15 06 41" src="https://user-images.githubusercontent.com/599268/185402585-41a6d129-0844-4acd-9dd1-f976e1ac9552.png"> | <img width="1438" alt="Screenshot 2022-08-18 at 15 04 01" src="https://user-images.githubusercontent.com/599268/185402583-e273b998-b196-44b0-9671-2dbd7986be89.png"> | <img width="1434" alt="Screenshot 2022-08-18 at 15 02 30" src="https://user-images.githubusercontent.com/599268/185402574-33419ad9-489e-4c20-bc11-5cee84010f52.png"> | <img width="1437" alt="Screenshot 2022-08-18 at 15 01 33" src="https://user-images.githubusercontent.com/599268/185402569-c211becc-864e-4897-9da7-ddad8d7e0c26.png"> | <img width="1437" alt="Screenshot 2022-08-18 at 15 01 01" src="https://user-images.githubusercontent.com/599268/185402566-181f7619-00cc-4d2e-947a-0b2f49275245.png"> |

Reply:
Perfect! ✨ 

Thanks, @jankeromnes! 🏀